src/sentry/spans/consumers/process/flusher.py

```diff
@@ -139,8 +155,41 @@ def poll(self) -> None:
         self.next_step.poll()
 
     def submit(self, message: Message[FilteredPayload | int]) -> None:
```

Comment:
Please add a short docstring that shows the two conditions we return with back pressure. Let's also document in which of the two cases the flusher itself will stop and why.